templates/index.html

```diff
@@ -22,9 +22,14 @@ <h3 class="mt-5">Previous runs</h3>
             {% for output in outputs %}
                 <div class="card {% if loop.first %}bg-primary{% endif %}" style="width: 25%;">
                     <div class="card-body">
-                        <h5 class="card-title">{{ output.date_display }}</h5>
-                        <a href="{{ url_for('output', path=output.html_path) }}" class="card-link">Animation</a>
-                        <a href="{{ url_for('output', path=output.log_path) }}" class="card-link">Logs</a>
+                        {% if output.is_valid() %}
```

Comment:
Should this instead be handled in `get_outputs` instead? This way we're not dupming out empty `div`s

Reply:
We aren't outputting empty divs? See the `else` -- we fall back to just displaying the directory name and an indication that it errored.